bgc_data/nrs_depth_binned_ctd_map.sql

```diff
@@ -11,7 +11,19 @@ WITH nrs_trips AS (
       FROM bgc_trip
       WHERE projectname = 'NRS'
 ),
---calculate the absolute time difference 
+--transform manually site code for VBM station
+dp_transformed AS (
```

Comment:
It would be a bit simpler to include this in the previous sub-query (`WITH nrs_trips AS ...`) -- i.e. set that site code to 'VBM100' when `stationcode='VBM'`.